app-modules/service-management/src/Filament/Resources/ServiceRequestResource/Pages/ViewServiceRequest.php

```diff
@@ -112,6 +113,17 @@ public function infolist(Infolist $infolist): Infolist
                                     Contact::class => ContactResource::getUrl('view', ['record' => $respondent->id]),
                                 };
                             }),
+                        TextEntry::make('time_to_resolution')
+                            ->label('Time to Resolution')
+                            ->formatStateUsing(function ($state) {
+                                $interval = Carbon::now()->diffAsCarbonInterval(Carbon::now()->addSeconds($state));
+                                $days = $interval->d;
+                                $hours = $interval->h;
+                                $minutes = $interval->i;
+
+                                return "{$days}d {$hours}h {$minutes}m";
+                            })
```

Comment:
You will need a `visible` check that checks a Feature Flag you create for this

Reply:
Done